GitUI/GitUICommands.cs

```diff
@@ -1485,10 +1485,10 @@ public bool StartRebase(IWin32Window owner, string onto)
                 interactive: false, startRebaseImmediately: true);
         }
 
-        public bool ContinueRebase(IWin32Window owner)
+        public bool StartContinueRebaseDialog(IWin32Window owner)
```

Comment:
I do not get the meaning with StartContinueRebase, is the meaning StartOrContinue or should not Start be there?

Reply:
it's: start the rebase dialog to continue rebase.
I'll rename it to `StartTheContinueRebaseDialog`